Deploy/UtilityScripts/CpqApiHelper.py

```diff
@@ -149,6 +219,13 @@ def getHeaderJwt(self):
         return {
             'Authorization': 'Bearer ' + self.__tokens['jwtToken']
         }
+    def getCTHeaderJwt(self):
+        return {"Authorization": f"Bearer {self.__tokens['jwtToken']}", "accept": "*/*"}
+    def postCTHeaderJwt(self):
+        return {"Authorization": f"Bearer {self.__tokens['jwtToken']}",
+                "accept": "*/*",
+                "Content-Type": "application/json"
+        }
```

Comment:
Like we said above, lets look at standardizing with the getHeaderJwt method and getHeaderBearer method.  We dont need two methods that do nearly the same thing.